lib/cylc/suite_logging.py

```diff
@@ -396,7 +396,7 @@ def _create_logs(self, log_logger_level=logging.INFO):
     def _register_syncronised_logs(self):
         """Establishes synchronisation between the logs."""
         self._group = RollingFileHandlerGroup()
-        for log in [self.loggers[log_name] for log_name in self.ALL_LOGS]:
+        for log in (self.loggers[log_name] for log_name in self.ALL_LOGS):
```

Comment:
Does this work? If so, it will be easier to read.

```python
for log_name in self.ALL_LOGS:
    self._group_add(self.loggers[log_name])
    # ...
```